.github/workflows/release-artifacts.yml

```diff
@@ -56,6 +56,15 @@ jobs:
           bin: qdrant
           target: ${{ matrix.target }}
           token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build Debian package
+        run: |
+          cargo install cargo-deb
+          cargo deb --no-build
+      - name: Upload Debian package
+        uses: actions/upload-artifact@v2
+        with:
+          name:  ${{ matrix.target }}-debian
```

Comment:
It looks like you are building it in `build-mac-binaries` for mac os. It should be in `build-linux-binaries`